cibuildwheel/windows.py

```diff
@@ -9,62 +10,90 @@
 IS_RUNNING_ON_TRAVIS = os.environ.get('TRAVIS_OS_NAME') == 'windows'
 
 
-def get_python_path(config):
-    nuget_args = get_nuget_args(config)
-    return os.path.join(nuget_args[-1], nuget_args[0] + "." + config.version, "tools")
+def simple_shell(args, env=None, cwd=None):
+    print('+ ' + ' '.join(args))
+    args = ['cmd', '/E:ON', '/V:ON', '/C'] + args
+    return subprocess.check_call(' '.join(args), env=env, cwd=cwd)
 
 
-def get_nuget_args(configuration):
-    python_name = "python" if configuration.version[0] == '3' else "python2"
-    if configuration.arch == "32":
-        python_name = python_name + "x86"
-    return [python_name, "-Version", configuration.version, "-OutputDirectory", "C:/cibw/python"]
+if IS_RUNNING_ON_AZURE or IS_RUNNING_ON_TRAVIS:
+    shell = simple_shell
+else:
+    run_with_env = os.path.abspath(os.path.join(os.path.dirname(__file__), 'resources', 'appveyor_run_with_env.cmd'))
+
+    # run_with_env is a cmd file that sets the right environment variables
+    # to build on AppVeyor.
+    def shell(args, env=None, cwd=None):
```

Comment:
hmm. I do not understand. Why one function is defined in if clause and second outside. And why use negative selection? Adding next CI for windows need to introduce next variable (ex. circleci has windows https://circleci.com/docs/2.0/hello-world-windows/). Maybe introduce variable `IS_RUNNING_ON_APPVEYOR`?

Reply:
This is how it was when the functions where nested. The one is `simple_shell` and the other one is called `shell`.
Why do I need `IS_RUNNING_ON_APPVEYOR`? We are already checking `IS_RUNNING_ON_AZURE` and `IS_RUNNING_ON_TRAVIS`. It has the same outcome.

But again, I did not change anything. This is just what was there before and what I moved out of `build`.

Reply:
Yeah, this snippet of code has grown to be a little weird. 

(I'm not even sure if that `appveyor_run_with_env.cmd` is needed any more, it might only fix up things for python 3.4 and lower... maybe i'll try removing it :P)

no need to change anything in this PR